src/currencyWallets/currency.test.js

```diff
@@ -7,10 +8,16 @@ import {
 } from '../test/fakeCurrency.js'
 import { fakeExchangePlugin } from '../test/fakeExchange.js'
 import { awaitState } from '../util/redux/reaction.js'
-import { assert } from 'chai'
+import { PRECISION } from './api'
+import chai from 'chai'
```

Comment:
Why not `import { assert } from 'chai'`? Or does the `chai-as-promised` mess this up?